tests/topotests/bgp_evpn_rt5/test_bgp_evpn.py

```diff
@@ -482,6 +517,90 @@ def test_evpn_ping_again():
     _test_evpn_ping_router(tgen.gears["r1"], ipv4_only=True)
 
 
+def test_evpn_enable_l3vpn_importation():
+    """
+    Reconfigure IPv6 networks
+    Unshutdown BGP L3VPN neighbors
+    Ensure R1 receives 10 EVPN prefixes
+    """
+    tgen = get_topogen()
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    tgen.gears["r2"].vtysh_cmd(
+        """
+        configure terminal\n
```

Comment:
Drop \n chars from every line, it's not necessary.

Reply:
ok